id-policy.md

```diff
@@ -0,0 +1,231 @@
+classes/terms, and the ontology itself) are uniquely identified by a
+[URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier). Within
+the OBO family of ontologies, we require the use of Permanent URLs
+(PURLs) with a standard base prefix.
+
+For example:
+
+ * [http://purl.obolibrary.org/obo/UBERON_0000955](http://purl.obolibrary.org/obo/UBERON_0000955) uberon class PURL for *brain*
+ * [http://purl.obolibrary.org/obo/obi.owl](http://purl.obolibrary.org/obo/obi.owl) ontology PURL for OBI
+
+Note that legacy formats such as obo-format and many bioinformatics
+databases require the use of short-form identifiers (e.g GO:0008150)
+to identify ontology terms. In this document we provide a well-defined
+mapping between these short prefixed identifiers and PURLs by
+considering the identifiers to be
+[CURIE](https://en.wikipedia.org/wiki/CURIE)s.
+
+This document addresses identifiers *only* for ontology terms, and not
+for Dbxrefs.
+
+## Design goals
+
+ - There must be a predictable, bidirectional mapping between OBO ids,
+and OBO Foundry-compliant URIs.
+ - The URIs should resolve to useful information about a term.
+ - The URIs should be designed so that they can be maintained over timeto keep pointing to useful information.
+ - Each OBO id is assigned to a only single term within the set of all OBO ontologies.
+ - There is a 1:1 mapping of OBO ids to Foundry-compliant URIs
+
+#### Format of OBO format ids
+
+See [OBO syntax](http://owlcollab.github.io/oboformat/doc/obo-syntax.html) for details.
+
+All OBO term IDs are CURIEs (prefixed identifiers) of the form `IDSpace:LocalId`
+
+Note that internally within an OBO Format document, relation IDs may
+be short form such as `part_of`, but these are mapped to CURIEs via the xref field.
+
+#### Format of Foundry-compliant URIs
+
+`FOUNDRY_OBO_URI ::= "http://purl.obolibrary.org/obo/" IDSPACE "_" LOCALID`
+
+#### Mapping of OWL ids to OBO format ids
+
+The recommended way is to use the [OBO JSON LD context](https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/registry/context.jsonld)
+
+This provides mappings between the OBO prefix and a base URL. See the [JSON-LD spec](https://json-ld.org/spec/latest/json-ld/) for a full specification of the expansion/contraction mechanism.
+
+CURIE to URI:
+
+`s@((**[A-Za-z_]***):(d+))@http://purl.obolibrary.org/obo/$1_$2@`
+
+URI to CURIE
+
+`s@http://purl.obolibrary.org/obo/(**[A-Za-z_]***)_(d+)@$1:$2@`
+
+
+Using these rules the OBO id**GO:0050918** is mapped to
+ the Foundry-compliant URI
+**http://purl.obolibrary.org/obo/GO_0050918** 
+
+## Policy for OBO Foundry ontologies
+
+All OBO ontologies should use PURLs that have a base URL within their allocated space (in CURIE form these would be prefixed with their ID space)
+
+
+## Response to Web requests for OBO URIs
+
+It is expected that the Foundry-compliant URIs behave, on the web,
+usefully. It will be the role of the OBO Foundry to supply generic
```

Comment:
Suggest rephrasing to present tense: "The OBO Foundry supplies generic..." with maybe link to Ontobee?

Reply:
+1